RelhaxModpack/RelhaxModpack/Windows/GameCenterUpdateDownloader.xaml

```diff
@@ -68,10 +68,10 @@
         <TabItem Header="Download Update Files" IsEnabled="False" x:Name="GcDownloadStep4">
             <Grid Background="#FFE5E5E5" Tag="GcDownloadStep4Gred">
                 <Button x:Name="GcDownloadStep4PreviousButton" HorizontalAlignment="Left" Margin="10,0,0,10" VerticalAlignment="Bottom" Width="75" Click="GcDownloadStep4PreviousButton_Click" Tag="GcDownloadStep4PreviousButton">
-                    <TextBlock x:Name="GcDownloadStep4Previous" Text="Previous" HorizontalAlignment="Stretch" VerticalAlignment="Stretch"/>
+                    <TextBlock x:Name="GcDownloadStep4PreviousText" Text="Previous" HorizontalAlignment="Stretch" VerticalAlignment="Stretch"/>
```

Comment:
Nice find. Consistency is key, but when you are "in the zone", sometimes you forget in your rush to get all the code from your brain to the screen 😄

Reply:
Thanks. Yea, I guess with all the V1 bits, some oldies, copy-paste leftovers and me coming to the scene pretty late, the best way would it to go top to bottom with such things, but that would take time plus I know that I would feel the urge to move things around within the Translation.cs file. I am the revolution over evolution type, which is not good for collaborations so I will hopefully hunt stuff down slowly and steadily :D